pkgs/by-name/me/memos/package.nix

```
@@ -24,6 +24,7 @@ let
 
   memos-protobuf-gen = stdenvNoCC.mkDerivation {
     name = "memos-protobuf-gen";
+    pname = "memos-protobuf-gen";
     inherit src;
```

I don't think it's recommended to use the `name` attribute anymore. Instead, we can do:

```suggestion
    pname = "memos-protobuf-gen";
    inherit version src;
```